### ISSUE TITLE
[BLUEJ-188] Cannot select second/subsequent fields in object inspector when Voiceover is on

### ISSUE THREAD

**Issue by neilccbrown** [k-pet-group/BlueJ-Greenfoot#188]
Mac OS X. When the Voiceover utility is switched on, it isn't possible to select fields in the object inspector.

---

**Issue metadata**

- Issue type: Bug
- Priority: Medium
- Fix versions: 3.0.0
